### PR TITLE
--build option for up command

### PR DESCRIPTION
**Pull request by kunalkushwaha**
Added --build option for `up` command
Partial fix for #207

Signed-off-by: Kunal Kushwaha <kushwaha_kunal_v7@lab.ntt.co.jp>

### CODE REVIEW

**Comment by joshwget** [comment]
LGTM

File: docker/service.go [hunk 1]
```diff
@@ -132,7 +132,11 @@ func (s *Service) collectContainers(ctx context.Context) ([]*Container, error) {
 	return result, nil
 }
 
-func (s *Service) ensureImageExists(ctx context.Context, noBuild bool) error {
+func (s *Service) ensureImageExists(ctx context.Context, noBuild bool, forceBuild bool) error {
```
Comment (vdemeester) [inline docker/service.go]: We might want to pass the `options` here instead of boolean arguments :angel:

Comment (vdemeester) [inline docker/service.go]: But that can be done in a follow-up though :angel:

**Comment by vdemeester** [comment]
LGTM 🐸

**Pull request merged**